app/assets/javascripts/services/alerts_center_service.js

```diff
@@ -532,7 +538,7 @@ function alertsCenterService(API, $q, $timeout, $document, $uibModal, $http) {
     }
 
     updateAlertStatus(newAlert);
-
+    providerLink(alertData.ems_id, newAlert);
```

Comment:
What about having the link straight to `'/restful_redirect/index?model=ExtManagementSystem&id=' + providerId` and just use `redirect_to` so it is consistent?